lib/private/Contacts/ContactsMenu/ContactsStore.php

```diff
@@ -284,8 +284,11 @@ public function findOne(IUser $user, int $shareType, string $shareWith): ?IEntry
 	private function contactArrayToEntry(array $contact): Entry {
 		$entry = new Entry();
 
-		if (isset($contact['id'])) {
-			$entry->setId($contact['id']);
+		if (isset($contact['UID'])) {
+			$entry->setId($contact['UID']);
+			$uid = $contact['UID'];
+			$avatar = $this->urlGenerator->linkToRoute('core.getAvatar', ['userId' => $uid, 'size' => 64]);
```

Comment:
@CarlSchwan Once i applied line 290 the contact with uploaded images disappeared from the menu and soon as i applied $avatar = "/index.php/avatar/$uid/64"; back they came back

Reply:
oh should probably be 'core.Avatar.getAvatar' instead

Reply:
Okay, ill give that a try and let you know! thanks :)

Reply:
No luck same results with $avatar = $this->urlGenerator->linkToRoute('core.Avatar.getAvatar', ['userId' => $uid, 'size' => 64]);

Reply:
Not sure if casing matters, also it should be an absolute URL
```suggestion
			$avatar = $this->urlGenerator->linkToRouteAbsolute('core.avatar.getAvatar', ['userId' => $uid, 'size' => 64]);
```